### ISSUE TITLE
yacd does not support custom log-level

### ISSUE THREAD

**Issue by emdoe** [haishanh/yacd#14]
No matter users choose which log mode on the `Config` panel (e.g., warning or silent), the `Logs`panel seems always displaying the logs in `info` mode. Indeed, yacd may ignore the user's setting of log-level in the config file.

**Comment by haishanh** [comment]
@emdoe There was an existing bu on this, but fixed already. Not sure how you are using yacd, make sure you are the latest one, e.g. thru http://yacd.haishan.me

Also note, changes made on the UI about clash config are not persisted to the clash config file.

**Comment by emdoe** [comment]
> @emdoe There was an existing bu on this, but fixed already. Not sure how you are using yacd, make sure you are the latest one, e.g. thru http://yacd.haishan.me

Yeah, I had read the old closed issue you mentioned. Unfortunately, it may not work properly as we desired. Here are the screenshots:
![config-panel](https://user-images.githubusercontent.com/2088116/57503839-92c65880-7324-11e9-880d-e7429f64950b.jpg)
![logs-panel](https://user-images.githubusercontent.com/2088116/57504001-43345c80-7325-11e9-8520-59030d91218d.jpg)


> Also note, changes made on the UI about clash config are not persisted to the clash config file.

Extremely thank you for the note though I know this. I would like to know could yacd reads the log-level setting from the config file (e.g., config.yml)?

Thanks for the prompt reply!

**Comment by haishanh** [comment]
@emdoe I did some tests myself, indeed I observed the same result as you said.

yacd use the `GET /logs` API to fetch logs from clash. The API is documented here https://clash.gitbook.io/doc/restful-api/common

It seems the `log-level` config in clash only has the effect to stdout(or/and stderr) of the clash, not to the `GET /logs` API. For example if you set the `log-level` to `silent`, the clash cli does not produce logs, but the `GET /logs` rest API still does.

Not sure if it's a bug of clash or clash is doing this intentionally.

**Comment by emdoe** [comment]
Thanks for reference clash's document! Interestingly, I find 'official' clash dashboard may also cancel this option on the settings panel (I think I had seen the option about log level before). Could yacd provide us an option to hide the `Logs` panel (or require a password to access this panel)?

**Comment by haishanh** [comment]
> Could yacd provide us an option to hide the Logs panel (or require a password to access this panel)?

No, that's not something I'd like to do.

I found clash's `GET /logs` API support adding a query string `level` to the url to indicating the which log level of logs you want to receive. For example, if we call the API with `GET /logs?level=error`, we will not see `info` level logs. I just published a version which will set the `level` param of the API url to the same value of prop "log-level" in the `GET /configs` response. I think this should be something you(and other users) are expected. 

Gonna close this issue.

Thanks.
